MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -126,14 +126,14 @@
 		.shaderInt8 = true,
 		.descriptorIndexing = _vulkan12FeaturesNoExt.descriptorIndexing,
 		.shaderInputAttachmentArrayDynamicIndexing = _metalFeatures.arrayOfTextures,
-		.shaderUniformTexelBufferArrayDynamicIndexing = _metalFeatures.arrayOfTextures,
-		.shaderStorageTexelBufferArrayDynamicIndexing = _metalFeatures.arrayOfTextures,
-		.shaderUniformBufferArrayNonUniformIndexing = false,
```

Comment:
What are the ramifications of indicating such complete support?

Will it cause other CTS tests, that check these flags, to fail if we actually don't support them?

Reply:
> I don't see why non-uniform indexing can't work.

Ah. Good point.